src/javascript/binary/base/utility.js

```diff
@@ -77,23 +78,52 @@ const getPropertyValue = (obj, k) => {
 const handleHash = () => {
     const hash = window.location.hash;
     if (hash) {
-        $(`a[href="${hash}"]`).click();
+        document.querySelector(`a[href="${hash}"]`).click();
     }
 };
 
 const clearable = (element) => {
     element.addClass('clear');
```

Comment:
here's another one :)

Reply:
afaik, this is from before and the element being passed to this function is a jquery element, so should be fine for now

Reply:
ok cool, how about line 86: `element.addClass('clear')` ?

Reply:
yes, that's what I thought we are discussing

Reply:
this is confusing because when this function was created element should have been named $element but it wasn't. it's part of the code from before and unchanged by this PR. it will be changed later when we get to files that are calling this function and we stop sending jquery elements to it.

Reply:
oh ya, i thought you were talking about line 80 there 😅

Reply:
otherwise, pr looks good 👍